### PR TITLE
Feat/migration

### PR DESCRIPTION
**Pull request by ShirinAI**
Add liquibase migration, remove dataLoader data

### CODE REVIEW

**Review by ignatIgnatov — COMMENTED** [review]
I have no experience with liquibase, but looks good. You can only delete unnecessary commented lines in DataLoader and DataLoaderConfig files.

**Comment by ignatIgnatov** [comment]
Resolve conflicts.

File: src/main/java/ludogorie_soft/reservations_platform_api/data/DataLoader.java [hunk 1]
```diff
@@ -14,20 +14,20 @@ public DataLoader(RoleRepository roleRepository) {
...
-            roleRepository.save(adminRole);
-        }
-
-        if (roleRepository.findByName("USER") == null) {
-            Role userRole = new Role();
-            userRole.setName("USER");
-            roleRepository.save(userRole);
-        }
-    }
+//    @PostConstruct
```
Comment (emiliyank) [inline src/main/java/ludogorie_soft/reservations_platform_api/data/DataLoader.java:17]: delete the code instead of commenting it

Comment (ShirinAI) [reply src/main/java/ludogorie_soft/reservations_platform_api/data/DataLoader.java]: Code deleted

**Review by emiliyank — APPROVED** [review]

**Pull request merged**